src/components/button/Button.tsx

```diff
@@ -32,11 +33,12 @@ type Props = {
   iconLeft?: any,
   iconRight?: any,
   iconOnly?: any,
+  iconButton?: any,
```

Comment:
Ik zou nog een betere naam zoeken dan iconButton. Waarmee verschilt dit van iconOnly? In dat het een iconOnly button is met een border? Misschien is dan iconOnlyWithBorder beter :)